Sources/Extensions/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -43,6 +43,43 @@ public extension Array where Element: FloatingPoint {
 	
 }
 
+// MARK: Methods (String)
+public extension Array where Element == String {
+    
+    /// SwifterSwift: Takes an array of strings and generate two lists:
+    ///               One is derived from their first character which will serve as their section label.
+    ///               Second is the array items now grouped based on their corresponding section (first letter).
+    ///
+    ///    ["Jason", "Alice", "Jack"].groupToSections() -> (["A", "J"], [["Alice"], ["Jack", "Jason"]])
+    ///
+    /// - Returns: A tuple consists of the sections list, and its grouped items.
+    public func groupToSections() -> ([String], [[String]]) {
```

Comment:
As for me the output tuples this looks like a dictionary [Character: [String]]()